server/data/helpers/questionsHelper.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../../config/dbConfig');
+
+module.exports = {
+	getQuestions: async (id) => {
+		const allQuestions = await db('questions').select('id', 'date');
+		const selectedQuestion = await db('questions')
+			.select(
```

Comment:
Can the select be omitted if we're getting all fields from the table?

Reply:
Oh good point, I'll make that change